### ISSUE TITLE
Centralize list of model formats in org.geppetto.core

### ISSUE THREAD

**Issue by adrianq** [openworm/org.geppetto#367]
Design to be done but basically org.geppetto.core will keep a list of available model formats. Each model interpreter, simulator and conversion service will register their model format and after that all of them will be using the model format object declared in org.geppetto.core. The main different with current approach is that Geppetto Model Format will only be declared in the core bundle.
@tarelli you win :)

**Comment by tarelli** [comment]
@adrianq what made you change your mind?:)

**Comment by adrianq** [comment]
Comparing ModelFormat (Enum) from different bundles (when looking for a conversion services for instance) can become a little bit tricky as toString() or name() its needed as far as I know. When you are looking inside a list or hashmap this is a pain. Anyway we need to go in to the details...

**Comment by adrianq** [comment]
TBH, this was the trigger. I see some other advantages about having this centralize in the core.

**Comment by adrianq** [comment]
Closing this issue. From now on, when the bundle is started it will register the model formats and, afterwards, the model interpreter, simulator and conversion service if needed. I had to change some tests which were testing these services. There was an error because they used some model formats in the junit test itself. Obviously, as the bundle is not started the model formats are not registered. I tried to implement an elegant solution using spring annotation as we have in the NeuronSimulatorServiceTest or some virgo abstract classes for testing so that the unit test initialises the bundle context. I couldn't make it work so I went for the quick and dirty deluxe solution. I register the model formats in the beforeclass method and then I executed the tests as usual. For instance:  [lems model interpreter tests] (https://github.com/openworm/org.geppetto.model.neuroml/blob/datamanager/src/test/java/org/geppetto/model/neuroml/test/LEMSModelInterpreterServiceTest.java#L69).
